database/seeders/CreateRoleSeeder.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\Role;
+use App\Models\UserRole;
+use App\Models\VendorEmployee;
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+
+class CreateRoleSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        Role::create([
+            'role'=>'Employee'
+        ]);
+        Role::create([
+            'role'=>'Vendor'
+        ]);
+        Role::create([
+            'role'=>'Employee'
+        ]);
+        Role::create([
+            'role'=>'Vendor'
```

Comment:
@Babita3331 Define an array of roles and may be loop through them to seed roles.